roles/federation/templates/kctestuser2.j2

```diff
@@ -0,0 +1,17 @@
+unset OS_CLOUD
+export OS_CACERT=/home/cloud-admin/full-ca-list.crt
+export OS_PROJECT_NAME="{{ cifmw_federation_project_name }}"
+export OS_PROJECT_DOMAIN_NAME="{{ cifmw_federation_keystone_domain }}"
+export OS_AUTH_URL="{{ cifmw_federation_keystone_url }}/v3"
+export OS_IDENTITY_API_VERSION=3
+export OS_AUTH_PLUGIN=openid
+export OS_AUTH_TYPE=v3oidcpassword
+export OS_USERNAME="{{ cifmw_federation_keycloak_testuser2_username }}"
```

Comment:
similar comment to the others.  Can't we just hard code all of these?  Templating these would only make sense if you had a single kctestuser.j2 file that you iterated through for the different users